flash_examples/generic_task.py

```diff
@@ -36,7 +36,7 @@
 )
 
 # 3. Load a dataset
-dataset = datasets.MNIST(os.path.join(_PATH_ROOT, 'data'), download=True, transform=transforms.ToTensor())
+dataset = datasets.MNIST(os.path.join(_PATH_ROOT, 'data'), download=False, transform=transforms.ToTensor())
```

Comment:
@tchaton @Borda was the issue with downloading solved ? because as a workaround we could just use CIFAR10 which has the same interface as MNIST

Reply:
We download the data just above, so there is no need to try to download them.